### PR TITLE
completed css-cascade.

### PR DESCRIPTION
**Pull request by chris-ys-kim**
Screen Shot 2021-05-13 at 11 29 44 AM
Screen Shot 2021-05-13 at 11 29 39 AM

### CODE REVIEW

File: css-cascade/specificity.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+.bio {
+  color: green;
+}
+
+#timbl {
+  color: red;
+}
+
+#verou {
```
Comment (Zhiwutian) [inline css-cascade/specificity.css:17]: These selectors should be in reverse to demonstrate that specificity will override source order.

**Review by Zhiwutian — APPROVED** [review]
approved with comment!

**Comment by chris-ys-kim** [comment]
Screen Shot 2021-05-13 at 10 43 49 AM
Screen Shot 2021-05-13 at 10 44 01 AM

**Comment by chris-ys-kim** [comment]
Screen Shot 2021-05-13 at 11 29 44 AM
Screen Shot 2021-05-13 at 11 29 39 AM

**Pull request merged**